src/Resources/contao/languages/en/tl_url_rewrite.xlf

```diff
@@ -0,0 +1,180 @@
+                <source>Activate/deactivate rewrite rule ID %s</source>
+            </trans-unit>
+
+            <trans-unit id="tl_url_rewrite.typeRef.basic.0">
+                <source>Basic</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.typeRef.basic.1">
+                <source>Allows to define the request matching using the basic Symfony routing features.</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.typeRef.expert.0">
+                <source>Expert</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.typeRef.expert.1">
+                <source>Allows to define the request condition using the &lt;a href=&quot;https://symfony.com/doc/current/components/expression_language.html&quot; target=&quot;_blank&quot;>Expression Language&lt;/a&gt;. For more information please &lt;a href=&quot;https://symfony.com/doc/current/routing/conditions.html&quot; target=&quot;_blank&quot;>visit this link&lt;/a&gt;.</source>
+            </trans-unit>
+
+            <trans-unit id="tl_url_rewrite.examples.0.0">
+                <source>Find address on Google Maps:</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.0.1">
+                <source>Type: basic
+Path restriction: find/{address}
+Response code: 303 See Other
+Response redirect URL: https://www.google.com/maps?q={address}
+---
+Result: domain.tld/find/Switzerland → https://www.google.com/maps?q=Switzerland</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.1.0">
+                <source>Redirect to a specific news entry:</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.1.1">
+                <source>Type: basic
+Path restriction: news/{news}
+Requirements: [news => \d+]
+Response code: 301 Moved Permanently
+Response redirect URL: {{news_url::{news}|absolute}}
+---
+Result: domain.tld/news/123 → domain.tld/news-reader/foobar-123.html
+Result: domain.tld/news/foobar → 404 Page Not Found</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.2.0">
+                <source>Rewrite legacy URLs with query string:</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.2.1">
+                <source>Type: expert
+Path restriction: home.php
+Request condition: context.getMethod() == 'GET' and request.query.has('page')
+Response code: 301 Moved Permanently
+Response redirect URL: {{link_url::{page}|absolute}}
+---
+Result: domain.tld/home.php?page=123 → domain.tld/foobar-123.html</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.3.0">
+                <source>Rewrite URLs including slashes (without query string) to new domain:</source>
+            </trans-unit>
+            <trans-unit id="tl_url_rewrite.examples.3.1">
+                <source>Type: basic
+Hosts restriction: [beispiel.de]
```

Comment:
We should use `domain.tld` here too to remain consistent :) Also in the `Result:` part, please ;)

Reply:
Yes, but the rule defines here `beispiel.de` as old domain via `Hosts restriction` - that's why `domain.tld` would be wrong. I could use it as target instead of example.com

Reply:
Yeah, let's just get rid of the German stuff in English descriptions :)

Reply:
I should have taken a Swiss example :-D - I updated the example!